pkgs/development/python-modules/python-mapnik/default.nix

```diff
@@ -135,6 +136,5 @@ buildPythonPackage rec {
     maintainers = [ ];
     homepage = "https://mapnik.org";
     license = licenses.lgpl21Plus;
-    broken = true; # At 2024-11-13, test_raster_warping fails.
```

Comment:
Thanks for unbreaking !

Reply:
You're welcome. I did not really investigate the failing test since it seems to work for my use-cases. However there already seem to be a lot of failing tests, that the maintainers of the program seem to not care about, so I guess it's probably fine.